File: sdks/node/examples/agent_with_mcp.ts
import { Agent } from "alith";

const agent = new Agent({
  model: "gpt-4",
  preamble:
    "You are a calculator here to help the user perform arithmetic operations. Use the tools provided to answer the user question.",
  mcpConfigPath: "servers_config.json",
});
console.log(await agent.prompt("Calculate 10 - 3"));

File: sdks/python/examples/agent_with_mcp.py
from alith import Agent

agent = Agent(
    model="gpt-4-mini",
    mcp_config_path="servers_config.json",
)
print(agent.prompt("Read the file: agent_with_mcp.py"))
print(agent.prompt("Read the content of file: agent_with_memory.py"))
